tools/mo/openvino/tools/mo/convert_impl.py

```diff
@@ -828,6 +818,8 @@ def show_mo_convert_help():
 
 
 def input_model_is_object(argv):
+    if 'input_model' not in argv:
```

Comment:
may be it is good to leave a comment here that there can be `--saved_model_dir`, or meta specific option which are only used to store paths to the input model.